ch2-git-basics/cloning.qmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Cloning an Existing Repository"
+---
+
+## Introduction
+
+This section will guide you through the process of cloning a GitHub repository. Cloning a repository means making a local copy of a project that exists on a "remote" DevOps server such as Axure DevOps, GitHub or GitLab.
+
+This enables you to make changes and test locally before pushing to be approved and merged with the main branch on GitHub.
+
+There are several ways to clone an existing repository. Below, you'll find two methods. The first uses Rstudio and the second uses Git Bash. Start with step 1, which is the same for both methods, and then decide which method to follow. Once you have successfully cloned, you should see the files appear in the location you have chosen to store your repository.
+
+## Step 1: Clone the Repository
```

Comment:
This feels more like "Get the repository clone link" rather then "Clone the repository"